src/activities/webhooks/Elsa.Activities.Webhooks/Handlers/EvictWorkflowRegistryDaletedCacheHandler.cs

```diff
@@ -0,0 +1,24 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Elsa.Caching;
+using Elsa.Services.Workflows;
+using Elsa.Webhooks.Events;
+using MediatR;
+
+namespace Elsa.Activities.Webhooks.Handlers
+{
+    public class EvictWorkflowRegistryDeletedCacheHandler : INotificationHandler<WebhookDefinitionDeleted>
```

Comment:
To reduce redundant code, let's merge this class with `EvictWorkflowRegistryCacheHandler` - it can implement both interfaces:

- `INotificationHandler<WebhookDefinitionSaved>`
- `INotificationHandler<WebhookDefinitionDeleted>`